src/pages/understand-stacks/running-regtest-node.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Running a regtest node
+description: Learn how to set up and run a regtest node
+icon: RegtestIcon
+duration: 15 minutes
+experience: beginners
+tags:
+  - tutorial
+images:
+  large: /images/cli.svg
+  sm: /images/cli.svg
+---
+
+## Introduction
```

Comment:
Is this content significantly different from the testnet steps? It feels like a duplicate to me after skimming through it. If we just change up a small piece of the whole tutorial, could we maybe simplify?

Reply:
There's definitely a lot of commonalities between regtest/testnet/mainnet. Ideally I would love to have a single page for these kind of docs where JS is used to hide variants for each network. Kind of like the [API docs](https://blockstack.github.io/stacks-blockchain-api/) where commands for each network are hidden in a collapsable table or dropdown of sorts.
![Screen Shot 2021-05-11 at 10 54 55 AM](https://user-images.githubusercontent.com/2747302/117837311-572e3400-b247-11eb-993b-bf71d7af9b37.png)

Reply:
I believe we should find a way to collapse these pages, I think that's outside the scope of this PR as I've heard that regtest docs are desperately needed. I will add an issue to the docs repo to track this idea, however.